1.1/spec/18-Annex-C.adoc

```diff
@@ -1,6 +1,149 @@
 === Annex C 
 === (informative)
 
+=== Best-practice: A GeoSPARQL-enabled triple store as a backend of an OGC web service
+
+This annex presents best practices for the creation of Linked Data backends to OGC webservices.
+
+==== C.1 Accessing geospatial features in a SPARQL endpoint
```

Comment:
Why "geospatial", not "spatial"? GeoSPARQL features are not inherently geographical.